clippy_lints/src/functions.rs

```diff
@@ -28,6 +28,22 @@ declare_clippy_lint! {
     "functions with too many arguments"
 }
 
+/// **What it does:** Checks for functions with a large amount of lines.
+///
+/// **Why is this bad?** Functions with a lot of lines are harder to understand
+/// due to having to look at a larger amount of code to understand what the
+/// function is doing. Consider splitting the body of the function into
+/// multiple functions.
+///
+/// **Known problems:** None.
+///
+/// ```
+declare_clippy_lint! {
+    pub TOO_MANY_LINES,
+    complexity,
```

Comment:
Can you make this a `pedantic` lint? I don't think we can get the community to agree on a threshold below 200 ;)